test/e2e/e2e_test.go

```diff
@@ -93,8 +98,35 @@ func createTestNamespace(client *kubernetes.Clientset, namespaceName string) (*v
 	return ns, err
 }
 
+func createServiceEndpoint(client *kubernetes.Clientset, podName, namespace string, labels map[string]string) error {
```

Comment:
Please use the metrics endpoint of the service ca operator rather than creating a new pod.

Reply:
pushed changes to use metrics endpoint. please take a look.